frontend/src/components/admin/AdminDashboard.js

```diff
@@ -27,30 +56,104 @@ export class AdminDashboard extends Component {
     downloadMineManagerHistory(this.state.mineNo, { window, document });
   };
 
+  compareMineName = (a, b) => {
+    if (a.mine_name < b.mine_name) {
+      return -1;
+    }
+    if (a.mine_name > b.mine_name) {
+      return 1;
+    }
+    return 0;
+  };
+
   render() {
     return (
       <div className="landing-page">
         <div className="landing-page__header">
-          <h1>Admin View</h1>
+          <Row>
+            <Col sm={22} md={14} lg={12}>
+              <h1>Admin View</h1>
+            </Col>
+          </Row>
         </div>
         <div className="landing-page__content">
-          <div className="tab__content">
-            <h2>Pull Mine Manager History</h2>
-            <Input
-              placeholder="Enter a Mine Number"
-              value={this.state.mineNo}
-              onChange={this.handleChange}
-              style={{ width: "50%", "max-width": "300px" }}
-            />
-            <Button onClick={this.handleDownload}>Download History</Button>
-          </div>
-          <div className="tab__content">
-            <MinespaceUserManagement />
-          </div>
+          <Tabs
+            activeKey={this.state.activeTab}
+            defaultActiveKey="summary"
+            onChange={this.handleTabChange}
+            size="large"
+            animated={{ inkBar: true, tabPane: false }}
+          >
+            <TabPane tab="Verified Mines" key="verifiedMines">
+              <div className="tab__content">
+                <div>
+                  <h4>{this.state.verifiedMines.length}&nbsp;Verified Mines</h4>
+                  <div>
+                    {this.state.verifiedMines.length > 0 && (
+                      <AdminVerifiedMinesList
+                        minesVerifiedStatusList={this.state.verifiedMines.sort(
+                          this.compareMineName
+                        )}
+                      />
+                    )}
+                  </div>
+                </div>
+              </div>
+            </TabPane>
+            <TabPane tab="Mines to be Checked" key="unverifiedMines">
+              <div className="tab__content">
+                <div>
+                  <h4>{this.state.unverifiedMines.length}&nbsp;Mines Needing Re-Verification</h4>
+                  <div>
+                    {this.state.unverifiedMines.length > 0 && (
+                      <AdminVerifiedMinesList
+                        minesVerifiedStatusList={this.state.unverifiedMines.sort(
```

Comment:
Same comment about sorting in the render method here

Reply:
changed.